FEXCore/Source/Interface/Core/JIT/JIT.cpp

```diff
@@ -868,6 +868,10 @@ CPUBackend::CompiledCode Arm64JITCore::CompileCode(uint64_t Entry, uint64_t Size
   auto JITBlockTail = GetCursorAddress<JITCodeTail*>();
   CursorIncrement(sizeof(JITCodeTail));
 
+  // Ensure SpinLockFutex is natively aligned
```

Comment:
Instead of adding two asserts and making JITCodeTail packed, why not simply unconditionally pad to `alignof(JITCodeTail)`?

Reply:
Doing that breaks instcountci and working around it is hard™.